core/domain/suggestion_services.py

```diff
@@ -165,6 +168,17 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def edit_suggestion(suggestion, change_object):
+    """Edits the submitted suggestion.
+
+    Args:
+        suggestion: Suggestion. The suggestion to be updated.
+        change_object: Change. Object of change in suugestion.
```

Comment:
is this the correct type for `change_object` here?

Reply:
I think yes

Reply:
![image](https://user-images.githubusercontent.com/16653571/55892165-6b6c5680-5bd3-11e9-81aa-2a32b8ce8c51.png)
I don't see any `Change` class in the codebase, can you please add a link where this class is defined?

Reply:
thanks, @DubeySandeep, I was wrong.
Actually the class of ``` change_object ``` is ``` ExplorationChange ```

Reply:
done

Reply:
is this `edit_sugesstion` method is only for the exploration? OR it's a general edit suggestion method?